js/apps/system/_admin/aardvark/APP/react/src/views/views/ViewSettingsReactView.jsx

```diff
@@ -25,129 +31,203 @@ const ViewSettingsReactView = ({ name }) => {
-                </th>
-                <th className="collectionTh">
-                  <Textbox type={'number'} disabled={!isAdminUser}
-                           value={getNumericFieldValue(formState.consolidationIntervalMsec)}
-                           onChange={getNumericFieldSetter('consolidationIntervalMsec', dispatch)}/>
-                </th>
-              </tr>
-
-              <tr className="tableRow" id="row_change-view-id">
-                <th className="collectionTh">
-                  ID:
-                </th>
-                <th className="collectionTh">
-                  <div className="modal-text" id="change-view-id">
-                    {formState.id}
-                  </div>
-                </th>
-              </tr>
-
-              <tr className="tableRow" id="row_change-view-globallyUniqueId">
-                <th className="collectionTh">
-                  Globally Unique ID:
-                </th>
-                <th className="collectionTh">
-                  <div className="modal-text" id="change-view-globallyUniqueId">
-                    {formState.globallyUniqueId}
-                  </div>
-                </th>
-              </tr>
-
-              <tr className="tableRow" id="row_change-view-type">
-                <th className="collectionTh">
-                  Type:
-                </th>
-                <th className="collectionTh">
-                  <div className="modal-text" id="change-view-type">
-                    {formState.type}
-                  </div>
-                </th>
-              </tr>
-              </tbody>
-            </table>
+  return (
+    <div className={"centralContent"} id={"content"}>
+      <div
+        id={"modal-dialog"}
+        className={"createModalDialog"}
+        tabIndex={-1}
+        role={"dialog"}
+        aria-labelledby={"myModalLabel"}
+        aria-hidden={"true"}
+      >
+        <div className="modal-body">
+          <div className={"tab-content"}>
+            <div className="tab-pane tab-pane-modal active" id="General">
+              <table style={{ margin: 30, marginLeft: 20 }}>
+                <tbody>
+                  <tr className="tableRow" id="row_change-view-name">
+                    <th className="collectionTh">Name*:</th>
+                    <th className="collectionTh">
+                      <Textbox
+                        type={"text"}
+                        value={formState.name}
+                        onChange={updateName}
+                        required={true}
+                        disabled={frontendConfig.isCluster || !isAdminUser}
+                      />
+                    </th>
+                    <th className="collectionTh">
+                      <ToolTip
+                        title="The View name (string, immutable)."
+                        setArrow={true}
+                      >
+                        <span className="arangoicon icon_arangodb_info"></span>
```

Comment:
Here's a mix of reformats + new code. Impossible to review like this.

Reply:
I think VSCode sold me out here again despite disabling Prettier.

Reply:
Done.